packages/gestalt-codemods/generic-codemods/helpers/codemodHelpers.js

```diff
@@ -0,0 +1,274 @@
+// @flow strict
+import {
+  type ApiType,
+  type Collection,
+  type FileType,
+  type JSCodeShift,
+  type JSXNodeType,
+  type ImportDeclarationType,
+  type NodePathType,
+  type ImportSpecifierType,
+  type JSXAttributeType,
+} from './codemodFlowtypes.js';
+
+type InitializeType = {|
+  api: ApiType,
+  file: FileType,
+|};
+
+/**
+ * initialize: Sets the boilerplate required to work with jscodeshift
+ */
+const initialize = ({ api, file }: InitializeType): { j: JSCodeShift, src: Collection } => {
+  const j = api.jscodeshift;
+  const src = j(file.source);
+  return { j, src };
+};
+
+type IsGestaltImportType = {| importDeclaration: ImportDeclarationType |};
+
+/**
+ * isGestaltImport: Validates a Gestalt import path returning true if it matches 'gestalt'.
+ * E.g. import { Box } from 'gestalt' // true
+ * E.g. import { Box } from 'gestaltExtensions/box' // false
+ */
+const isGestaltImport = ({ importDeclaration }: IsGestaltImportType): boolean =>
```

Comment:
Some of these helper APIs seem a little verbose…do we ever expect `isGestaltImport` to have more than one argument? It seems simpler to use the arg directly so callsites can be more like `isGestaltImport(foo)` instead of `isGestaltImport({ importDeclaration: foo })`

Reply:
OOO. 

For consistency, I'll keep as is

Reply:
Also, the key informs what needs to be passed as well.

Reply:
Consistency doesn't need to be dogmatic, particularly since this collection of helpers is quite diverse. This particular case isn't a hill I'll die on, but not every helper function needs to look identical.

Reply:
Personally, I like named arguments even for this cases, as it can be more explicit what needs to be passed. what does need to be passed to isGestaltImport? the importDeclaration. It's more explicit and readable.